src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -1,10 +1,12 @@
+import {
```

Comment:
TS fails on 

https://github.com/algolia/instantsearch.js/pull/3940/files#diff-571e9df908c4237e29ced537895105b7R92

but I don't know what the solution is here, making the `getRefinements` function accept any doesn't really make sense to me